content/rooms/v11.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Room Version 11
+type: docs
+weight: 100
+---
+
+This room version builds on [version 10](/rooms/v10) while clarifying redaction
+rules.
+
+## Client considerations
+
+Clients should no longer depend on the `creator` property in the `content` of
+[`m.room.create`](/client-server-api#mroomcreate) events.  In all room versions,
+clients can rely on `sender` instead to determine a room creator.
+
+Clients should note that the format of [`m.room.redaction`](/client-server-api#mroomredaction)
+events has been modified and look for the `redacts` key under `content` instead
+of a top-level event property.
+
+Clients should note that the `third_party_invite` key of [`m.room.member`](/client-server-api#mroommember)
+events is no longer redacted, *but* will only contain the `signed` key after redaction.
```

Comment:
The way this is structured still makes me feel like this is an introduction, and that the real changes are going to be spelt out elsewhere.

Two things would help:

1. Add an "Event format" subheading.
2. Delete "Clients should note that".